app/src/main/kotlin/com/softteco/template/utils/FlowExtension.kt

```diff
@@ -26,7 +27,40 @@ inline fun <T1, T2, T3, T4, T5, T6, R> combine(
             args[2] as T3,
             args[3] as T4,
             args[4] as T5,
-            args[5] as T6,
+            args[5] as T6
         )
     }
 }
+
+@Suppress("MagicNumber")
+inline fun <T1, T2, T3, T4, T5, T6, T7, R> combine(
```

Comment:
If we add this function, the previous one becomes redundant, since this function takes 6 parameters out of the box.
But I'm not sure that this is necessary. We shouldn't just extend our combine when we need a new argument for our states.